building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+# Building a Preview
+
+- [Build a Preview](#build-a-preview)
+- [Share your Preview](#share-your-preview)
+- [Preview Actions](#preview-actions)
+- [Preview status](#preview-status)
+- [Set a Base Preview](#set-a-base-preview)
+- [Administer Base Previews](#administer-base-previews)
+- [Auto-generate Previews](#auto-generate-previews)
+- [Auto-delete Previews](#auto-delete-previews)
+- [Optimize your Preview builds](#optimize-your-preview-builds)
+- [Preview size explained](#preview-size-explained)
+
+## Build a Preview
+
+Once you've
+[set up your Tugboat project](../setting-up-tugboat/index.md#create-a-project),
+[linked a repo](../setting-up-tugboat/index.md#add-repos-to-the-project), and
+[put your Config file in the linked repo](../setting-up-tugboat/index.md#create-a-tugboat-config-file),
+it's time to build a Preview!
+
+- [How to build a Preview](#how-to-build-a-preview)
+- [The build process: explained](#the-build-process-explained)
+
+### How to build a Preview
+
+To build a Preview:
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to build a Preview;
+4. Click the name of the repo that contains the code you want to use to build
+   the Preview;
+5. Scroll down to the **Available to Build** section; by default, you'll see
+   _Branches_, but you can switch to view Tags or Pull Requests that are
+   available to Preview;
+6. Press the **Build Preview** button to begin the build.
+
+![Build preview](_images/available_to_build.png)
+
+While the Preview is building, you'll see the Preview appear in the **Latest
+Previews** section, with a yellow status indicator _building_.
+
+![Preview building](_images/preview_building.png)
+
+When the Preview is ready, the Preview button will turn blue, and you'll see a
+green status indicator _ready_. Simply press the Preview button to view the
+Preview. While you're at it, go ahead and
+[share your Preview](#share-your-preview) - we know you're proud of your work!
```

Comment:
> While you're at it, go ahead and
 [share your Preview](#share-your-preview) - we know you're proud of your work!

Do we want to have this link here? It scrolls the page down, and and there's not really a way to "go back to where I just was, there was more to that section I wasn't done reading". It was a little disorienting for me.

Reply:
I put that link there because there's a lot of concept info after that, and users might want to just skip to the next part - i.e. "I've built the Preview successfully, I don't need all this other stuff, I just want to find out how to share it." I'm still pondering how best to organize this page and the Setting up Services page - maybe it would make sense to completely separate the various processes and concept content, so the logical next point in the user's workflow here would be Share, and the concept stuff lives somewhere else. That might alleviate this issue.